train.py

```diff
@@ -118,6 +118,11 @@ def train(hyp, opt, device, tb_writer=None, wandb=None):
     scheduler = lr_scheduler.LambdaLR(optimizer, lr_lambda=lf)
     # plot_lr_scheduler(optimizer, scheduler, epochs)
 
+    # Logging
+    if wandb and wandb.run is None:
+        id = ckpt.get('wandb_id') if 'ckpt' in locals() else None
+        wandb_run = wandb.init(config=opt, resume="allow", project=log_dir, id=id)
+
```

Comment:
@glenn-jocher `project=log_dir` throws error. It's posixPath but a string is expected. 
And another catch is that the project name cannot contain '\' so you might wanna extract only the last part of the path which is the name of the run.
Maybe something like this:
```
project=str(os.path.basename(log_dir))
```

If you want to check this for yourself, you can execute this [colab](https://colab.research.google.com/drive/1e4K905OYm3jrAsmwD6LuAV19KDm0PuSf?usp=sharing) within 2 mins. If everything is fine, this should automatically log data to wandb
```

Reply:
Yes, you are correct! basename is a good solution. I'll push a commit.

Reply:
Ok I've pushed a bug fix for this. basename also converts the posixpath to a string. The only failure mode is is if logdir='', and name='', then project='.', but this would be extremely rare.

Reply:
trying the notebook now

Reply:
Yeah. These should be very rare and we can address them when they get reported. I'm running the test colab and it's working as expected.
Btw, have you had a chance to look at a this [doc](https://www.notion.so/wandbai/YoloV5-ec06daa32df44b6bb918d26759fdbb8f) that we sent you? You might wanna add it after merging the PR.